aws/resource_aws_codebuild_project.go

```diff
@@ -178,7 +177,6 @@ func resourceAwsCodeBuildProject() *schema.Resource {
 						"environment_variable": {
 							Type:     schema.TypeList,
 							Optional: true,
-							Computed: true,
```

Comment:
@bflad this helped `TestAccAWSCodeBuildProject_Environment_EnvironmentVariable` pass I'm not sure if this is the best change; in the original test, updating from 2 env variables -> 0 failed but 1->2 passed 😕